addon/-private/engine-instance-ext.js

```diff
@@ -230,21 +234,36 @@ EngineInstance.reopen({
     });
   },
 
+  getExternalRoute(routeName) {
```

Comment:
Hmm, there are many other methods here without an `_` prefix.  I guess lets leave this for now, and make a new issue to underscore any new methods to these core classes...